benches/uint.rs

```diff
@@ -219,7 +219,19 @@ fn bench_division(c: &mut Criterion) {
     let mut rng = make_rng();
     let mut group = c.benchmark_group("wrapping ops");
 
-    group.bench_function("div/rem, U256/U128, full size", |b| {
+    group.bench_function("div/rem, U256/U128", |b| {
+        b.iter_batched(
+            || {
+                let x = U256::random(&mut rng);
+                let y = U128::random(&mut rng);
+                (x, NonZero::new(y).unwrap())
+            },
+            |(x, y)| black_box(x.div_rem(&y)),
```

Comment:
Just a side node, you don't need to wrap the output in `black_box`, Criterion already does that for all `iter*` methods.